packages/react/src/patterns/blocks/LogoGrid/README.md

```diff
@@ -30,12 +30,12 @@ const title =
 const logosGroup = [
   {
     title: 'Company A',
-    imgSrc: 'https://via.placeholder.com/140',
+    imgSrc: 'https://dummyimage.com/140x140',
     altText: 'placeholder',
   },
   {
     title: 'Company B',
-    imgSrc: 'https://via.placeholder.com/140',
+    imgSrc: 'https://dummyimage.com/140x140',
```

Comment:
Styling and ratio?

Reply:
I didn't bother to put that in this part of the README, as I'd have to document the service's API. If you think it should be in the docs we can add a few lines for usage guidance.